### PR TITLE
Small updates (aiidalab, vm_hostname, vm_version, travis)

### PR DESCRIPTION
**Pull request by borellim**
- Use aiidalab ansible role v1.1.4 (aiidalab pip package 20.06.0b1).
- Change `vm_hostname` to `quantum-mobile`.
- Change `vm_version` to `20.06.0`.
- Slightly improve Travis checks (copied over from QM Desktop Edition).

### CODE REVIEW

File: globalconfig.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # Caution: This file is read by vagrant, ansible and bash
 vm_name: "Quantum Mobile"
-vm_version: "20.05.0"
+vm_version: "20.06.0"
```
Comment (chrisjsewell) [inline globalconfig.yml:4]: @ltalirz just want to clarify the protocol here for version naming: `<year>.<major>.<minor>`?

Comment (giovannipizzi) [reply globalconfig.yml:4]: I think it's `<year>.<month>.<minor>`, right @ltalirz ?

Comment (ltalirz) [reply globalconfig.yml:4]: yes

**Review by ltalirz — APPROVED** [review]
thanks @borellim !

**Pull request merged**